dip-0008.md

```diff
@@ -181,56 +152,101 @@ message is called `CLSIG` and has the following structure:
-is received by a node, it should ensure that only this block is locally
-accepted as the next block.
+This message is propagated through the inventory system.
 
-If an alternative block for the same height is received, it must be invalidated
-and removed from the currently active chain since a signed block has already
-been received. If the correct block is already present locally, its chain
-should be activated as the new active chain. If the correct block is not known
-locally, it must wait for this block to arrive and request it from other nodes
-if necessary.
+## Handling of signed blocks
+
+The node should ensure that the fully singed block is locally accepted as the
+next block. If the correct block is already present locally, its chain should
+be activated as the new active chain tip. If the correct block is not known
+locally, the node must wait for this block to arrive and request it from other
+nodes if necessary. If an alternative block for the same height is received,
+it must not be added to the currently active chain, if the correct block is
+already present locally.
 
 If a block has been received locally and no `CLSIG` message has been received
 yet, it should be handled the same way it was handled before the introduction
-of ChainLocks. This means the longest-chain and first-seen rules must be
+of ChainLocks. This means the most-work and first-seen rules must be
 applied. When the `CLSIG` message for this (or another) block is later
-received, the above logic must be applied.
-
-## Conflicting successful signing attempts
-
-While the network is operating as expected, it’s not possible to encounter
-two conflicting recovered signatures for two signing attempts of the same
-parent block. It is possible for a malicious masternode operator to manually
-double-sign two different attempts when a close race between two competing
-blocks occurs. If one of the conflicting signature shares is withheld until the
-second attempt succeeds and the conflicting signature is then propagated to the
-network, the two attempts will result in two valid recovered signatures.
-
-When performing the finalization of successful attempts, the LLMQ members will
-only try to finalize a single attempt, which is usually the first one to
-succeed. Only a single attempt will be able to gain a majority during
-finalization, which removes the possibility of conflicts. In the worst case,
-finalization completely fails, no `CLSIG` message is created and nodes must
-fall back to the first-seen and longest-chain rules.
+received, the below logic must be applied.
+
+### Handling of blocks signed via a single LLMQ
+
+Upon receipt, each node must perform the following verification before
+announcing it to other nodes:
+
+  1. Based on the deterministic masternode list at the given height,
+  a set of active quorums must be calculated.
+  2. A request id must be calculated.
+  3. A quorum must be selected from the set based on the calculated request id.
```

Comment:
```suggestion
  1. Based on the deterministic masternode list at the given height,
  the set of active quorums must be calculated.
  2. The request id must be calculated.
  3. The quorum must be selected from the set based on the calculated request id.
```

Might be only me but `A/a` sounds like there could be multiple but there can only be one of those for the specific height, no?